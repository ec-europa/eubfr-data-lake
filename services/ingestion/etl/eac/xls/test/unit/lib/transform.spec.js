/**
 * @jest-environment node
 */

import mapper from '../../../src/lib/transform';
import testRecord from '../../stubs/record.json';
import testRecord2 from '../../stubs/record2.json';
import testRecordFullYear from '../../stubs/record_full_year.json';
import testRecordCulture20072013ProjectsOverview from '../../stubs/record_Culture_2007_2013_Projects_Overview.json';

describe('DG EAC XLS transformer', () => {
  let yearTwoDigits = {};
  let typeWellFormated = {};
  let yearFourDigits = {};
  let cultureProjectsOverview = {};

  beforeAll(() => {
    yearTwoDigits = mapper(testRecord);
    typeWellFormated = mapper(testRecord2);
    yearFourDigits = mapper(testRecordFullYear);
    cultureProjectsOverview = mapper(testRecordCulture20072013ProjectsOverview);
  });

  test('Returns null when record is not provided', () => {
    expect(mapper()).toBe(null);
  });

  test('Produces correct JSON output structure, year in 2 digits', () => {
    expect(yearTwoDigits).toMatchSnapshot();
  });

  test('Produces correct JSON output structure, type array well formated', () => {
    expect(typeWellFormated).toMatchSnapshot();
  });

  test('Produces correct JSON output structure, year in 4 digits', () => {
    expect(yearFourDigits).toMatchSnapshot();
  });

  test('Produces correct JSON output structure, given Culture_2007_2013_Projects_Overview_2018-11-22 file from http://ec.europa.eu/programmes/creative-europe/projects/ce-projects-compendium/', () => {
    expect(cultureProjectsOverview).toMatchSnapshot();
  });
});
