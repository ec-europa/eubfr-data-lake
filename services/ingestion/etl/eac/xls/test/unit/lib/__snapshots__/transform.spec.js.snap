// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DG EAC XLS transformer Produces correct JSON output structure, given Culture_2007_2013_Projects_Overview_2018-11-22 file from http://ec.europa.eu/programmes/creative-europe/projects/ce-projects-compendium/ 1`] = `
Object {
  "action": "",
  "budget": Object {
    "eu_contrib": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
    "funding_area": Array [],
    "mmf_heading": "",
    "other_contrib": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
    "private_fund": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
    "public_fund": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
    "total_cost": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
  },
  "call_year": "",
  "complete": true,
  "description": "The project aims at restoring the European dimension of the Romani language and culture, to inscrease solidarity and encourage minority valorisation and defend their human rights through audiovisual media.  It aims at providing a wider range of information about the functioniong of minority self-governments, ministries' programmes, governmental organisations helping minoriries and handicap people: with a focus on the Rom heritage.
Keja will be in charge of film-making as didactic material and of technical trainings/interventions, and of providing linguistic adaptations of the audiovisual activities. A series of DVDs will be produced (with short movies, interactive exercises, recorded musical and literary illustrative material), a book, and a dictionary of the Romani language.",
  "ec_priorities": Array [],
  "media": Array [],
  "programme_name": "Culture\\\\Culture (2007-2013)\\\\Multi-annual Cooperation Projects",
  "project_id": "EACEA-148838",
  "project_locations": Array [
    Object {
      "address": "",
      "centroid": null,
      "country_code": "HU",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "BG",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "FR",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "PL",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "RO",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "SK",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
  ],
  "project_website": "",
  "related_links": Array [],
  "reporting_organisation": "EAC",
  "results": Object {
    "available": "No",
    "result": "https://ec.europa.eu/programmes/creative-europe/projects/ce-project-details/#project/148838-CU-1-2009-1-HU-CULTURE-VOL11",
  },
  "status": "",
  "sub_programme_name": "",
  "success_story": "No",
  "themes": Array [],
  "third_parties": Array [
    Object {
      "address": "Ülloi ut 23,  1091,  BUDAPEST",
      "country": "HU",
      "email": "",
      "name": "KISEBBSEGI ES EMBERI JOGI ALAPITVANY- MINORITY AND HUMAN RIGHTS FOUNDATION",
      "phone": "",
      "region": "",
      "role": "coordinator",
      "type": "",
      "website": "www.keja.hu",
    },
    Object {
      "address": "15 Noemvri Str.1   1040  SOFIA",
      "country": "BG",
      "email": "",
      "name": "INSTITUTE OF SOCIOLOGY AT BULGARIAN ACADEMY OF SCIENCES",
      "phone": "",
      "region": undefined,
      "role": "partner",
      "type": undefined,
      "website": "",
    },
    Object {
      "address": "RUE DE LILLE 2   75343  PARIS CEDEX 07",
      "country": "FR",
      "email": "",
      "name": "INSTITUT NATIONAL DES LANGUES ET CIVILISATIONS ORIENTALES",
      "phone": "",
      "region": undefined,
      "role": "partner",
      "type": undefined,
      "website": "",
    },
    Object {
      "address": "Rynek 20-21   33 100  TARNOW",
      "country": "PL",
      "email": "",
      "name": "REGIONAL MUSEUM IN TARNOW",
      "phone": "",
      "region": undefined,
      "role": "partner",
      "type": undefined,
      "website": "",
    },
    Object {
      "address": "Str.Buzesti 19, district 1   011011  BUCAREST",
      "country": "RO",
      "email": "",
      "name": "ROMANI CRISS - ROMA CENTRE FOR SOCIAL INTERVENTION AND STUDIES",
      "phone": "",
      "region": undefined,
      "role": "partner",
      "type": undefined,
      "website": "",
    },
    Object {
      "address": "Povazska ulica 20   04011  KOSICE",
      "country": "SK",
      "email": "",
      "name": "NADACIA DOBRA ROMSKA VILA KESAJ",
      "phone": "",
      "region": undefined,
      "role": "partner",
      "type": undefined,
      "website": "",
    },
  ],
  "timeframe": Object {
    "from": "2009-01-09T00:00:00.000Z",
    "from_precision": "day",
    "to": "2016-06-06T00:00:00.000Z",
    "to_precision": "day",
  },
  "title": "Restoring the European Dimension of the Romani Language & Culture",
  "type": Array [],
}
`;

exports[`DG EAC XLS transformer Produces correct JSON output structure, type array well formated 1`] = `
Object {
  "action": "Cooperation projects",
  "budget": Object {
    "eu_contrib": Object {
      "currency": "EUR",
      "raw": "200,000.00",
      "value": 200000,
    },
    "funding_area": Array [],
    "mmf_heading": "",
    "other_contrib": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
    "private_fund": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
    "public_fund": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
    "total_cost": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
  },
  "call_year": "2014",
  "complete": true,
  "description": "One of the biggest current challenges of Europe is the increasing youth unemployment rates. Among young artists, these rates can be even higher. At the same time, these new talents are the ones who are able to offer creative and innovative perspectives on economy, society and culture and are the ones to evoke dialogue about our moral parameters. Eight festivals, spread all over Europe and in their core dedicated to support emerging performing artists, have joined forces in the project International Young Makers in Action. Our vision is to strengthen and renew the European performing arts sector by creating opportunities for emerging artists. Our passion is to have emerging performing artists reaching all corners of Europe to have an impact on audiences, peers and professionals. With the unique focus on European emerging artists from all performing disciplines, IYMA will organize the following activities:• Presentation of at least four international performances by emerging artists per festival, adding up to a total of 60. • 15 contextualization programmes for audiences to embed the work in the local context and to offer possibilities for artists and audiences to meet.• 15 in-depth programmes for participating artists to develop their skills, deepen their knowledge, get inspired and broaden their horizon. • 15 presenters’ programmes for international professionals to scout work, get acquainted with artists and actively participate in network meetings and showcases. • 1 LAB WEEK where ten emerging artists explore artistic ideas and methodologies together through workshops, master classes and meetings.All activities will be followed by a group of bloggers. Alongside the activities an audience research will be done on a European level. With the unique combination of the international mobility of artists and an increased connectivity with audiences, peers and professionals, IYMA will offer a valuable contribution to the European performing arts landscape.",
  "ec_priorities": Array [],
  "media": Array [],
  "programme_name": "Creative Europe",
  "project_id": "552237-CREA-1-2014-1-NL-CULT-COOP1",
  "project_locations": Array [
    Object {
      "address": "",
      "centroid": null,
      "country_code": "NL",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "ES",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "IT",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "BA",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "PL",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "GB",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
  ],
  "project_website": "",
  "related_links": Array [],
  "reporting_organisation": "EAC",
  "results": Object {
    "available": "No",
    "result": "http://ec.europa.eu/programmes/creative-europe/projects/ce-project-details-page/?nodeRef=workspace://SpacesStore/b0cdaa31-b137-4471-a347-3cfbb8c1ea61",
  },
  "status": "finalized",
  "sub_programme_name": "Culture",
  "success_story": "No",
  "themes": Array [],
  "third_parties": Array [
    Object {
      "address": "TOLHUISWEG 2, 1031 CL, AMSTERDAM",
      "country": "NL",
      "email": "",
      "name": "STICHTING ITS FESTIVAL",
      "phone": "",
      "region": "Noord-Holland",
      "role": "coordinator",
      "type": "Festival (non Audiovisual)",
      "website": "",
    },
    Object {
      "address": "ARAUTI 5 181, 48902, BARAKALDO",
      "country": "ES",
      "email": "",
      "name": "TEATRAKO TEATRO ELKARTEA",
      "phone": "",
      "region": "Extra-Regio NUTS 2",
      "role": "partner",
      "type": "Festival (non Audiovisual)",
      "website": "www.actfestival.com",
    },
  ],
  "timeframe": Object {
    "from": "2014-10-09T00:00:00.000Z",
    "from_precision": "day",
    "to": "2016-10-08T00:00:00.000Z",
    "to_precision": "day",
  },
  "title": "International Young Makers in Action",
  "type": Array [
    "Fashion design",
    "Other",
    "Theatre",
    "Circus arts",
  ],
}
`;

exports[`DG EAC XLS transformer Produces correct JSON output structure, year in 2 digits 1`] = `
Object {
  "action": "Cooperation projects",
  "budget": Object {
    "eu_contrib": Object {
      "currency": "EUR",
      "raw": "200,000.00",
      "value": 200000,
    },
    "funding_area": Array [],
    "mmf_heading": "",
    "other_contrib": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
    "private_fund": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
    "public_fund": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
    "total_cost": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
  },
  "call_year": "2014",
  "complete": true,
  "description": "One of the biggest current challenges of Europe is the increasing youth unemployment rates. Among young artists, these rates can be even higher. At the same time, these new talents are the ones who are able to offer creative and innovative perspectives on economy, society and culture and are the ones to evoke dialogue about our moral parameters. Eight festivals, spread all over Europe and in their core dedicated to support emerging performing artists, have joined forces in the project International Young Makers in Action. Our vision is to strengthen and renew the European performing arts sector by creating opportunities for emerging artists. Our passion is to have emerging performing artists reaching all corners of Europe to have an impact on audiences, peers and professionals. With the unique focus on European emerging artists from all performing disciplines, IYMA will organize the following activities:• Presentation of at least four international performances by emerging artists per festival, adding up to a total of 60. • 15 contextualization programmes for audiences to embed the work in the local context and to offer possibilities for artists and audiences to meet.• 15 in-depth programmes for participating artists to develop their skills, deepen their knowledge, get inspired and broaden their horizon. • 15 presenters’ programmes for international professionals to scout work, get acquainted with artists and actively participate in network meetings and showcases. • 1 LAB WEEK where ten emerging artists explore artistic ideas and methodologies together through workshops, master classes and meetings.All activities will be followed by a group of bloggers. Alongside the activities an audience research will be done on a European level. With the unique combination of the international mobility of artists and an increased connectivity with audiences, peers and professionals, IYMA will offer a valuable contribution to the European performing arts landscape.",
  "ec_priorities": Array [],
  "media": Array [],
  "programme_name": "Creative Europe",
  "project_id": "552237-CREA-1-2014-1-NL-CULT-COOP1",
  "project_locations": Array [
    Object {
      "address": "",
      "centroid": null,
      "country_code": "NL",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "ES",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "IT",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "BA",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "PL",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "GB",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
  ],
  "project_website": "",
  "related_links": Array [],
  "reporting_organisation": "EAC",
  "results": Object {
    "available": "No",
    "result": "http://ec.europa.eu/programmes/creative-europe/projects/ce-project-details-page/?nodeRef=workspace://SpacesStore/b0cdaa31-b137-4471-a347-3cfbb8c1ea61",
  },
  "status": "finalized",
  "sub_programme_name": "Culture",
  "success_story": "No",
  "themes": Array [],
  "third_parties": Array [
    Object {
      "address": "TOLHUISWEG 2, 1031 CL, AMSTERDAM",
      "country": "NL",
      "email": "",
      "name": "STICHTING ITS FESTIVAL",
      "phone": "",
      "region": "Noord-Holland",
      "role": "coordinator",
      "type": "Festival (non Audiovisual)",
      "website": "",
    },
    Object {
      "address": "ARAUTI 5 181, 48902, BARAKALDO",
      "country": "ES",
      "email": "",
      "name": "TEATRAKO TEATRO ELKARTEA",
      "phone": "",
      "region": "Extra-Regio NUTS 2",
      "role": "partner",
      "type": "Festival (non Audiovisual)",
      "website": "www.actfestival.com",
    },
    Object {
      "address": "VIA BUONARROTI 30, 00185, ROMA",
      "country": "IT",
      "email": "",
      "name": "ASSOCIAZIONE CULTURALE AREA06",
      "phone": "",
      "region": "Lazio",
      "role": "partner",
      "type": "Art association",
      "website": "www.shorttheatre.org",
    },
    Object {
      "address": "MARSALA TITA 54 / 1, 71000, SARAJEVO",
      "country": "BA",
      "email": "",
      "name": "JAVNA USTANOVA MES-MEDJUNARODNI TEATARSKI FESTIVAL",
      "phone": "",
      "region": "Bosnia And Herzegovina",
      "role": "partner",
      "type": "Festival (non Audiovisual)",
      "website": "",
    },
    Object {
      "address": "UL MIODOWA 22/24, 00 246, WARSZAWA",
      "country": "PL",
      "email": "",
      "name": "AKADEMIA TEATRALNA IM. ALEKSANDRA ZELWEROWICZA W WARSZAWIE",
      "phone": "",
      "region": "Mazowieckie",
      "role": "partner",
      "type": "Higher education institution (tertiary level)",
      "website": "www.at.edu.pl",
    },
    Object {
      "address": "BROAD STREET CENTENARY SQUARE, B1 2EP, BIRMINGHAM",
      "country": "UK",
      "email": "",
      "name": "BE FESTIVAL CIC",
      "phone": "",
      "region": "West Midlands",
      "role": "partner",
      "type": "Festival (non Audiovisual)",
      "website": "www.bestival.org",
    },
  ],
  "timeframe": Object {
    "from": "2014-10-09T00:00:00.000Z",
    "from_precision": "day",
    "to": "2016-10-08T00:00:00.000Z",
    "to_precision": "day",
  },
  "title": "International Young Makers in Action",
  "type": Array [],
}
`;

exports[`DG EAC XLS transformer Produces correct JSON output structure, year in 4 digits 1`] = `
Object {
  "action": "Cooperation projects",
  "budget": Object {
    "eu_contrib": Object {
      "currency": "EUR",
      "raw": "200,000.00",
      "value": 200000,
    },
    "funding_area": Array [],
    "mmf_heading": "",
    "other_contrib": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
    "private_fund": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
    "public_fund": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
    "total_cost": Object {
      "currency": "",
      "raw": "",
      "value": 0,
    },
  },
  "call_year": "2014",
  "complete": true,
  "description": "One of the biggest current challenges of Europe is the increasing youth unemployment rates. Among young artists, these rates can be even higher. At the same time, these new talents are the ones who are able to offer creative and innovative perspectives on economy, society and culture and are the ones to evoke dialogue about our moral parameters. Eight festivals, spread all over Europe and in their core dedicated to support emerging performing artists, have joined forces in the project International Young Makers in Action. Our vision is to strengthen and renew the European performing arts sector by creating opportunities for emerging artists. Our passion is to have emerging performing artists reaching all corners of Europe to have an impact on audiences, peers and professionals. With the unique focus on European emerging artists from all performing disciplines, IYMA will organize the following activities:• Presentation of at least four international performances by emerging artists per festival, adding up to a total of 60. • 15 contextualization programmes for audiences to embed the work in the local context and to offer possibilities for artists and audiences to meet.• 15 in-depth programmes for participating artists to develop their skills, deepen their knowledge, get inspired and broaden their horizon. • 15 presenters’ programmes for international professionals to scout work, get acquainted with artists and actively participate in network meetings and showcases. • 1 LAB WEEK where ten emerging artists explore artistic ideas and methodologies together through workshops, master classes and meetings.All activities will be followed by a group of bloggers. Alongside the activities an audience research will be done on a European level. With the unique combination of the international mobility of artists and an increased connectivity with audiences, peers and professionals, IYMA will offer a valuable contribution to the European performing arts landscape.",
  "ec_priorities": Array [],
  "media": Array [],
  "programme_name": "Creative Europe",
  "project_id": "552237-CREA-1-2014-1-NL-CULT-COOP1",
  "project_locations": Array [
    Object {
      "address": "",
      "centroid": null,
      "country_code": "NL",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "ES",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "IT",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "BA",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "PL",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
    Object {
      "address": "",
      "centroid": null,
      "country_code": "GB",
      "location": null,
      "nuts": Array [],
      "postal_code": "",
      "region": "",
      "town": "",
    },
  ],
  "project_website": "",
  "related_links": Array [],
  "reporting_organisation": "EAC",
  "results": Object {
    "available": "No",
    "result": "http://ec.europa.eu/programmes/creative-europe/projects/ce-project-details-page/?nodeRef=workspace://SpacesStore/b0cdaa31-b137-4471-a347-3cfbb8c1ea61",
  },
  "status": "finalized",
  "sub_programme_name": "Culture",
  "success_story": "No",
  "themes": Array [],
  "third_parties": Array [
    Object {
      "address": "TOLHUISWEG 2, 1031 CL, AMSTERDAM",
      "country": "NL",
      "email": "",
      "name": "STICHTING ITS FESTIVAL",
      "phone": "",
      "region": "Noord-Holland",
      "role": "coordinator",
      "type": "Festival (non Audiovisual)",
      "website": "",
    },
    Object {
      "address": "ARAUTI 5 181, 48902, BARAKALDO",
      "country": "ES",
      "email": "",
      "name": "TEATRAKO TEATRO ELKARTEA",
      "phone": "",
      "region": "Extra-Regio NUTS 2",
      "role": "partner",
      "type": "Festival (non Audiovisual)",
      "website": "www.actfestival.com",
    },
    Object {
      "address": "VIA BUONARROTI 30, 00185, ROMA",
      "country": "IT",
      "email": "",
      "name": "ASSOCIAZIONE CULTURALE AREA06",
      "phone": "",
      "region": "Lazio",
      "role": "partner",
      "type": "Art association",
      "website": "www.shorttheatre.org",
    },
    Object {
      "address": "MARSALA TITA 54 / 1, 71000, SARAJEVO",
      "country": "BA",
      "email": "",
      "name": "JAVNA USTANOVA MES-MEDJUNARODNI TEATARSKI FESTIVAL",
      "phone": "",
      "region": "Bosnia And Herzegovina",
      "role": "partner",
      "type": "Festival (non Audiovisual)",
      "website": "",
    },
    Object {
      "address": "UL MIODOWA 22/24, 00 246, WARSZAWA",
      "country": "PL",
      "email": "",
      "name": "AKADEMIA TEATRALNA IM. ALEKSANDRA ZELWEROWICZA W WARSZAWIE",
      "phone": "",
      "region": "Mazowieckie",
      "role": "partner",
      "type": "Higher education institution (tertiary level)",
      "website": "www.at.edu.pl",
    },
    Object {
      "address": "BROAD STREET CENTENARY SQUARE, B1 2EP, BIRMINGHAM",
      "country": "UK",
      "email": "",
      "name": "BE FESTIVAL CIC",
      "phone": "",
      "region": "West Midlands",
      "role": "partner",
      "type": "Festival (non Audiovisual)",
      "website": "www.bestival.org",
    },
  ],
  "timeframe": Object {
    "from": "2014-10-09T00:00:00.000Z",
    "from_precision": "day",
    "to": "2016-10-08T00:00:00.000Z",
    "to_precision": "day",
  },
  "title": "International Young Makers in Action",
  "type": Array [],
}
`;
